### PR TITLE
PROD-854#2157-fix-Activation is not sending when platform is activated in Multisite setup

### LINKED ISSUE #2157: Activation is not sending when platform is activated in Multisite setup
**To Reproduce**
1. Activate the BuddyBoss Platform in any subsite
2. Go to Admin Dashboard > Users > Add New
3. Add a new user and **DO NOT** skip confirmation email
4. Notice that the user you added does not receive any confirmation email

**Expected behavior**
The user should receive a confirmation Email just like when the platform is deactivated

**Screenshots**
new user

**Support ticket links**
https://secure.helpscout.net/conversation/1421786505/124479
https://secure.helpscout.net/conversation/1518305453/143099

**Jira issue** : [PROD-854]

[PROD-854]: https://buddyboss.atlassian.net/browse/PROD-854?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### PR DESCRIPTION
**Pull request by meetmakadiya**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #2157

### How to test the changes in this Pull Request:

1. Activate the BuddyBoss Platform in any subsite
2. Go to Admin Dashboard > Users > Add New
3. Add a new user and DO NOT skip confirmation email
4. Notice that the user you added does not receive any confirmation email

### Proof Screenshots or Video

http://i.imgur.com/ISAOoYJ.png

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

**Jira issue** : [PROD-854]

[PROD-854]: https://buddyboss.atlassian.net/browse/PROD-854?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dipskakadiya — CHANGES_REQUESTED** [review]
@meetmakadiya Can you check by activating the plugin at the network level? I think else condition is required in some cases.

**Comment by meetmakadiya** [comment]
@dipskakadiya  due to `%` on sprintf so through the error and `$message` variable blank so mail is not sent so updated the template.

File: src/bp-templates/bp-nouveau/buddypress/assets/emails/wp/email-template.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
 defined( 'ABSPATH' ) || exit;
 
 $settings = bp_email_get_appearance_settings();
-$width = wp_is_mobile() ? '100%' : '600px';
+$width = wp_is_mobile() ? '100%%' : '600px';
```
Comment (dipskakadiya) [inline src/bp-templates/bp-nouveau/buddypress/assets/emails/wp/email-template.php:46]: @meetmakadiya Can you run PHPCS formating on this?

**Review by dipskakadiya — CHANGES_REQUESTED** [review]

File: src/bp-templates/bp-nouveau/buddypress/assets/emails/wp/email-template.php (hunk 1 above)
Comment (meetmakadiya) [reply src/bp-templates/bp-nouveau/buddypress/assets/emails/wp/email-template.php:46]: @dipskakadiya yes I run the PHPCS and have not found any kind of issue.
[report.txt](https://github.com/buddyboss/buddyboss-platform/files/7576754/report.txt)

**Review by dipskakadiya — APPROVED** [review]

**Pull request closed without merging**